doc/deprecated.rst

```diff
@@ -143,6 +143,9 @@ Miscellaneous
 * As of Twig 1.x, ``Twig_Environment::clearTemplateCache()`` is deprecated and
   will be removed in 2.0.
 
+* As of Twig 1.x, ``Twig_Environment::clearCacheFiles()`` is deprecated and
```

Comment:
I would say ``as of Twig 1.22``. Deprecations are always added in a specific release. ``As of 1.x`` does not mean anything